test/tests/userobjects/nearest_point_layered_side_flux_average/nearest_point_layered_side_flux_average.i

```diff
@@ -0,0 +1,79 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 40
+  ny = 10
+  nz = 10
```

Comment:
this is too much. Your gold file is half a megabyte.
3D is ok but with 100x less points at most

Reply:
Will reduce the size.